scripts/core/ui/ui.ts

```diff
@@ -842,11 +842,21 @@ function splitterWidget(superdesk, $timeout, $rootScope) {
             const MONITORING_MIN_WIDTH = 400;
             const AUTHORING_MIN_WIDTH = 730;
 
-            const workspace = element,
-                authoring = element.next('#authoring-container'),
+            let workspace, authoring, container;
+
+            const initializeContainers = () => {
+                workspace = element ? element :
+                    angular.element('#workspace-container');
+
+                authoring = element.next('#authoring-container');
```

Comment:
Why are you using `.next()` to get an element by ID? Since IDs are unique, wouldn't `angular.element('#authoring-container')` would achieve the same?

Reply:
I was thinking that next should search for smaller amount of files (because it will always be next to it) and `angular.element` will go through complete DOM

Reply:
Getting an element by ID is very fast, you certainly don't need to worry about performance here. I doubt that `angular.element` would check every node. It is probably using native browser methods under the hood like `document.getElementById` / `document.querySelector` which are optimized by the browser.